AdyenDropIn/DropInComponent.swift

```diff
@@ -127,7 +127,7 @@ public final class DropInComponent: NSObject, PresentableComponent {
     
     // MARK: - Private
 
-    private lazy var componentManager = createComponentManager(order, nil)
+    private lazy var componentManager = createComponentManager(nil, nil)
```

Comment:
I was looking at this line as well :D 
we can make it `private var componentManager: ComponentManager` and move initiation to init